app/scripts/views/settings/avatar_change.js

```diff
@@ -87,6 +87,8 @@ function ($, Cocktail, FormView, AvatarMixin, SettingsMixin, Template,
       var self = this;
       var defer = p.defer();
       var file = e.target.files[0];
+      var account = self.getSignedInAccount();
+      self.logAccountImageChange(account);
```

Comment:
If the user cancels out of the file picker, will this still record a change event? Should it?